io.openems.edge.battery.fenecon.home/src/io/openems/edge/battery/fenecon/home/FeneconHomeBatteryImpl.java

```diff
@@ -294,111 +330,286 @@ protected ModbusProtocol defineModbusProtocol() throws OpenemsException {
+										.bit(2, generateBcuChannel(i, "SYSTEM_AFE_OVER_VOLTAGE")) //
+										.bit(3, generateBcuChannel(i, "SYSTEM_AFE_UNDER_VOLTAGE")) //
+										.bit(4, generateBcuChannel(i, "SYSTEM_HIGH_TEMPERATURE_PERMANENT_FAILURE")) //
+										.bit(5, generateBcuChannel(i, "SYSTEM_LOW_TEMPERATURE_PERMANENT_FAILURE")) //
+										.bit(6, generateBcuChannel(i, "SYSTEM_HIGH_CELL_VOLTAGE_PERMANENT_FAILURE")) //
+										.bit(7, generateBcuChannel(i, "SYSTEM_LOW_CELL_VOLTAGE_PERMANENT_FAILURE")) //
+										.bit(8, generateBcuChannel(i, "SYSTEM_SHORT_CIRCUIT"))), //
+								m(generateBcuChannel(i, "_SOC"), new UnsignedWordElement(towerOffset + 8), // [%]
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), //
+								m(generateBcuChannel(i, "_SOH"), new UnsignedWordElement(towerOffset + 9), // [%]
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), //
+								m(generateBcuChannel(i, "_VOLTAGE"), new UnsignedWordElement(towerOffset + 10), // [V]
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), //
+								m(generateBcuChannel(i, "_CURRENT"), new UnsignedWordElement(towerOffset + 11), // [A]
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), //
+								m(generateBcuChannel(i, "_MIN_CELL_VOLTAGE"),
+										new UnsignedWordElement(towerOffset + 12)), // [mV]
+								m(generateBcuChannel(i, "_MAX_CELL_VOLTAGE"),
+										new UnsignedWordElement(towerOffset + 13)), // [mV]
+								m(generateBcuChannel(i, "_AVARAGE_CELL_VOLTAGE"),
+										new UnsignedWordElement(towerOffset + 14)), //
+								m(generateBcuChannel(i, "_MAX_CHARGE_CURRENT"),
+										new UnsignedWordElement(towerOffset + 15)), //
+								m(generateBcuChannel(i, "_MIN_CHARGE_CURRENT"),
+										new UnsignedWordElement(towerOffset + 16)), //
+								m(generateBcuChannel(i, "_BMS_SERIAL_NUMBER"),
+										new UnsignedWordElement(towerOffset + 17)), //
+								m(generateBcuChannel(i, "_NO_OF_CYCLES"), new UnsignedWordElement(towerOffset + 18)), //
+								m(new UnsignedWordElement(towerOffset + 19)) //
+										.m(generateBcuChannel(i, "_DESIGN_CAPACITY"),
+												ElementToChannelConverter.SCALE_FACTOR_MINUS_1) // [Ah]
+										.m(Battery.ChannelId.CAPACITY, ElementToChannelConverter.DIRECT_1_TO_1) // [%]
+										.build(), //
+								m(generateBcuChannel(i, "_USABLE_CAPACITY"), new UnsignedWordElement(towerOffset + 20), //
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), // [Ah]
+								m(generateBcuChannel(i, "_REMAINING_CAPACITY"),
+										new UnsignedWordElement(towerOffset + 21), //
+										ElementToChannelConverter.SCALE_FACTOR_MINUS_1), // [Ah]
+								m(generateBcuChannel(i, "_MAX_CELL_VOLTAGE_LIMIT"),
+										new UnsignedWordElement(towerOffset + 22)), //
+								m(generateBcuChannel(i, "_MIN_CELL_VOLTAGE_LIMIT"),
+										new UnsignedWordElement(towerOffset + 23))));
+			}
+		} catch (OpenemsException e) {
+			this.log.info("Dynamic Channels could not created");
```

Comment:
This should never happen. Add `e.printstacktrace()` so that it becomes for visible in the log.